docs/core-concepts/packages.md

```diff
@@ -0,0 +1,170 @@
+
+The `netstandard1.3` framework is a package-based target framework, so it doesn't expose or describe any APIs in the absence of packages that target that framework. It is the interplay of the framework and the packages that define the set of APIs that are available when targeting the framework.
+
+Package-based Target Frameworks
+===============================
+
+There is a two-way relationship between frameworks and packages. The first part is defining the APIs available for a given framework, for example `netstandard1.3`. Continuing with that example, there are no APIs that are inherently part of `netstandard1.3`. All of the `netstandard1.3` APIs come from packages that target `netstandard1.3` or compatible frameworks, such as `netstandard1.0`. These packages and the APIs they expose define the framework.
+
+The second part of the relationship is asset selection. Packages can contain assets for multiple frameworks. Given a reference to a set of packages and/or metapackages, the target framework is needed to determine which asset should be selected, for example `net46` or `netstandard1.3`. It is important to select the correct asset. For example, a `net46` asset is not likely to be compatible with .NET Framework 4.0 or .NET Core 1.0.
+
+It is an interesting question of where a package-based target framework's definition ends and where consumption of that definition starts. There isn't a crisp answer for that. The answer is a function of a given project.json file. Your dependencies create your view of the framework, independent of the publisher(s) of those dependencies. This means that we can consider the definition of `netstandard` to be greater than the libraries that are nominally part of .NET Core or shipped by Microsoft.
+
+The two primary package-based target frameworks used with .NET Core are:
+
+- `netstandard`
+- `netcoreapp`
+
+.NET Standard
+-------------
+
+The .NET Standard (TFM: `netstandard`) framework represents the APIs defined by and built on top of the [.NET Standard Library](../concepts/dotnet-standard-library.md). Libraries that are intended to run on multiple runtimes should use this target. They will be supported on any .NET Standard compliant runtime, such as .NET Core, .NET Framework and Mono/Xamarin. Each of these runtimes supports a set of .NET Standard versions, depending on which APIs they implement. 
+
+The `NETStandard.Library` metapackage targets the `netstandard` framework. The most common way to target `netstandard` is by referencing this metapackage. It describes and provides access to the ~40 .NET  libraries and associated APIs that define the .NET Standard Library. You can reference additional packages that target `netstandard` to get access to additional APIs. These higher-level packages can be thought of as "targeting netstandard" or as "a .NET Standard Library" but they do not define "the .NET Standard Library".
```

Comment:
> These higher-level packages can be thought of as "targeting netstandard" or as "a .NET Standard Library" but they do not define "the .NET Standard Library".

I don't think this is really a problem that can be solved with documentation, but it's *really* confusing to read that when I depend on something called `NETStandard.Library` and I target something called `netstandard` I'm not targeting "the .NET Standard Library".  It reads to me that there is no such thing as "the .NET Standard Library", but rather many .NET Standard Libraries.  This is conflicting with just about anything else I read which describes things as "the .NET Standard Library".